src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -156,4 +156,22 @@ export class ScriptProjectCreateStep extends ProjectCreateStepBase {
             await fse.writeFile(funcIgnorePath, this.funcignore.sort().join(os.EOL));
         }
     }
+
+    private getHostContent(wizardContext: IProjectWizardContext, runtime: ProjectRuntime): object {
+        if (runtime === ProjectRuntime.v2) {
+            if (wizardContext.language === ProjectLanguage.PowerShell) {
```

Comment:
We really shouldn't check the language... What we should have is either a Map or List of languages that _support_ managed dependencies.

I know that PowerShell is the only supported language, but it would be more readable and future proofing to invest in that now.

That said, if @EricJizbaMSFT is fine with this for now, I won't block on this.

However, I will block on at least adding a comment that says this is for managed dependencies.

Reply:
Yeah I would suggest you add a property to this class:
```
protected supportsManagedDependencies: boolean = false;
```

And then in `PowerShellProjectCreateStep.ts` overwrite it with:
```
protected supportsManagedDependencies: boolean = true;
```

Reply:
@EricJizbaMSFT: Thanks for the suggestion. I've updated the code accordingly.